R/groups.R

```diff
@@ -117,7 +117,7 @@ group_keys.polars_data_frame <- function(.tbl, ...) {
     if (is_polars_lf(out)) {
       out <- out$collect()
     }
-    out$to_data_frame()
+    as.data.frame(out)
```

Comment:
Sinse dplyr depends on tibble, how about use `as_tibble` instead?

And, the line `out$collect()` does not seem needed. (`as_tibble(<lazyframe>) calls $collect`)

Reply:
indeed, thanks